modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/edit_category.jsp

```diff
@@ -67,7 +67,7 @@ if (portletTitleBasedNavigation) {
 
 <div <%= portletTitleBasedNavigation ? "class=\"container-fluid container-fluid-max-xl container-form-lg\"" : StringPool.BLANK %>>
 	<c:if test="<%= !portletTitleBasedNavigation %>">
-		<h3><%= mbHomeDisplayContext.getTitle() %></h3>
+		<h3><liferay-ui:message key="<%= mbHomeDisplayContext.getTitle() %>" /></h3>
```

Comment:
Hi @Ankita288 ,
Can you please do the translation in https://github.com/liferay/liferay-portal/blob/master/modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/display/context/DefaultMBHomeDisplayContext.java#L36 ?
That method already has a translated return value, so if that one is returned the liferay-ui:message tag is not needed